instructor/function_calls.py

```diff
@@ -276,10 +276,13 @@ def from_response(
     @classmethod
     def parse_cohere_json_schema(
         cls: type[BaseModel],
-        completion: NonStreamedChatResponse,
+        completion: Any,
         validation_context: Optional[dict[str, Any]] = None,
         strict: Optional[bool] = None,
     ):
+        from cohere import NonStreamedChatResponse
+
+        assert isinstance(completion, NonStreamedChatResponse)
```

Comment:
```suggestion
        assert isinstance(completion, NonStreamedChatResponse), "completion is not of type `NonStreamedChatResponse`"
```

Reply:
but maybe better to just check if hasattr("text")

Reply:
converted to 

```
 assert hasattr(
            completion, "text"
        ), "Completion is not of type NonStreamedChatResponse"
```